testing/fuchsia/run_tests.py

```diff
@@ -102,18 +104,31 @@ def bundled_test_runner_of(target_id: str) -> BundledTestRunner:
           tests
       )
   )
+  packages = []
   for test in tests:
-    original_package = test['package']
-    test['package'] = os.path.join(
-        OUT_DIR, test['package'].replace('-0.far', '.far')
-    )
-    try:
-      os.remove(test['package'])
-    except FileNotFoundError:
-      pass
-    os.symlink(original_package, test['package'])
+    if 'package' in test:
+      packages.append(test['package'])
+    else:
+      assert 'packages' in test, \
+             'Expect either one package or a list of packages'
+      packages.extend(test['packages'])
+  resolved_packages = []
+  for package in set(packages):
+    if package.endswith('-0.far'):
+      # Make a symbolic link to match the name of the package itself without the
+      # '-0.far' suffix.
+      original_package = package
+      package = os.path.join(OUT_DIR, package.replace('-0.far', '.far'))
+      try:
+        os.remove(package)
+      except FileNotFoundError:
+        pass
+      os.symlink(original_package, package)
+      resolved_packages.append(package)
+    else:
+      resolved_packages.append(os.path.join(OUT_DIR, package))
```

Comment:
This logic would be clearer with some different variable names:

```
if package.endswith('-0.far'):
   new_package_path = os.path.join(OUT_DIR, package.replace('-0.far', '.far'))
   try:
      os.remove(new_package_path)
   except FileNotFoundError:
      pass
   os.symlink(package_path, new_package_path)
   resolved_packages.add(new_package_path)
else:
   resolved_packages.add(os.path.join(OUT_DIR, package))
```

Reply:
Also it's not immediately clear to me why we need to remove the new_package_name file (in case it already exists? Is it possible for it to ever exist?) Maybe a comment would be useful here.

Reply:
Yes, if running locally, the symbolic link won't be removed automatically. Added comment.